README.md

```diff
@@ -511,6 +511,20 @@ usersRes.data.forEach((user) => {
 });
 ```
 
+#### Set or Expire User Password
+
+You can set or expire a user's password.
+Note: When setting a password, it will be initially set as expired.
+The user could not log-in an expired password, and must replace it.
```

Comment:
```suggestion
Note: When setting a password, it will automatically be set as expired.
The user will not be able log-in using an expired password, and will be required replace it.
```